### ISSUE TITLE
Improve print form functionality

### ISSUE THREAD

**Issue by QAZIMAAZARSHAD** [QAZIMAAZARSHAD/Voters-Registration-Portal#246]
Not working on mobile.

Many sections are not printing on the desktop either.

**Comment by RiyaGupta89** [comment]
Is this issue closed?

**Comment by QAZIMAAZARSHAD** [comment]
> Is this issue closed?

No, this one is open, the previous issue was not resolved completely but I awarded you points for a few improvements.


It is yet to be completed, now make PR's mentioning this issue.

**Comment by RiyaGupta89** [comment]
@QAZIMAAZARSHAD But I'm not able to understand the issue. 
![print](https://user-images.githubusercontent.com/72425181/123362438-4c78f580-d58e-11eb-923f-687e5bd254f3.png)
The details are getting printed even in the mobile view. I don't know what is the issue now that needs to be resolved.

**Comment by QAZIMAAZARSHAD** [comment]
I tried again, details are not printing on mobile.

https://qazimaazarshad.github.io/Voters-Registration-Portal/ open link in mobile and see.
